### PR TITLE
Add static_context attribute to StreamV2

### PR DESCRIPTION
**Pull request by sgallagher**
This PR depends on #520 

Only the final patch in the set is new.

### CODE REVIEW

**Comment by sgallagher** [comment]
Don't review yet; I haven't implemented the YAML parser and emitter changes yet.

**Comment by sgallagher** [comment]
Should be ready to review now.

File: modulemd/modulemd-module-stream-v2.c [hunk 1]
```diff
@@ -2856,6 +2987,10 @@ modulemd_module_stream_v2_emit_yaml (ModulemdModuleStreamV2 *self,
       return FALSE;
     }
 
+  if (modulemd_module_stream_v2_is_static_context (self))
+    {
+    }
```
Comment (mmathesius) [inline modulemd/modulemd-module-stream-v2.c:2992]: I'm only reviewing the last commit at the moment. Looks like the emit code didn't make it into this commit. Some specific tests for parsing/emitting static_context would be in order--which would have caught this. Everything else that's here looks good too me!

Comment (sgallagher) [reply modulemd/modulemd-module-stream-v2.c:2993]: Whoops! :anguished: Yeah, I think I got distracted and forgot to add the emit. Thanks for catching that.

I've added a test as well now.

**Review by mmathesius — APPROVED** [review]
The last commit looks good to me now. Thanks!

**Pull request merged**